platform/ios/Integration Tests/MGLTestLocationManager.m

```diff
@@ -0,0 +1,51 @@
+#import "MGLTestLocationManager.h"
+
+// Used to supply integration tests with a simulated location manager.
+// Methods that are empty are not used within integration tests and are
+// therefore unimplemented.
+
+@implementation MGLTestLocationManager
+
+@synthesize delegate;
+
+- (instancetype)init
```

Comment:
This `init` method isn't needed here since it doesn't do anything in addition to `super`.